src/openms/source/ANALYSIS/TOPDOWN/FLASHDeconvAlgorithm.cpp

```diff
@@ -1237,9 +1244,9 @@ namespace OpenMS
       }
 
 #ifdef _OPENMP
-  #pragma omp for schedule(static) ordered
+#pragma omp for schedule(static) ordered
       for(int i=0; i<omp_get_num_threads(); i++) {
-  #pragma omp ordered
+#pragma omp ordered
```

Comment:
block below could be
```
#else
      filtered_peak_groups = std::move(filtered_peak_groups_private);
#endif
```
?

Reply:
I think this is OK as is. It is not a bottleneck. Furthermore, when I tried different omp instructions, it generated a few unexpected results (e.g., the number of peak groups change per run or crash). And as we want to make this windows safe, we cannot use good omp keywrods like "task" So this is the best I can do. This part gives "a little bit" of speed boost. In the future maybe I can optimize this, but I do not think it would be really necessary.